src/ResourceManager/ApiManagement/Commands.ApiManagement/help/Add-AzureRmApiManagementRegion.md

```diff
@@ -117,7 +117,7 @@ Valid values are:
 Type: System.Nullable`1[Microsoft.Azure.Commands.ApiManagement.Models.PsApiManagementSku]
 Parameter Sets: (All)
 Aliases:
-Accepted values: Developer, Standard, Premium, Basic
```

Comment:
Why are these accepted values being removed? I still see them here: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/ApiManagement/Commands.ApiManagement/Models/PsApiManagementSku.cs

I see this also in all the files below (I went through the first couple but assume it's generally true) - why are they being removed?

Reply:
Looks like some issue with platyPS module, I just ran the command again 
`Update-MarkdownHelp -Path D:\github\azure-powershell\src\ResourceManager\ApiManagement\Commands.ApiManagement\help -UseFullTypeName -AlphabeticParamsOrder`

Reply:
That's very odd.  Did you do a full build before running the update-markdownhelp command?  If that doesn't fix it, can you just manually revert? We don't want bad information getting there, and I can investigate the platyPS issue after you get this PR in.

Reply:
reverted using git, rerunning Import-Module after clean build did not help. 

---
In reply to: [203885613](https://github.com/Azure/azure-powershell/pull/6697#discussion_r203885613) [](ancestors = 203885613)